composer.json

```diff
@@ -20,9 +20,9 @@
     },
     "require-dev": {
         "atoum/atoum": "~2.0",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/yaml": "~2.3|~3.0",
-        "symfony/console": "~2.3|~3.0",
```

Comment:
i prefer using `2.7` as the lowest constraints because `2.7` is LTS and add Symfony 4 constraints

Reply:
okay